components/tools/OmeroWeb/omeroweb/webadmin/webadmin_utils.py

```diff
@@ -3,6 +3,9 @@
 import logging
 
 from django.core.validators import validate_email
+from django.conf import settings
+
+from omero.util.upgrade_check import UpgradeCheck
```

Comment:
Please move this back into `upgradeCheck()`. The scoped import was done to not leak `omero` packages into `webadmin_utils` so that it could be imported without `omero` on the `PYTHONPATH`.